docs/examples/SPLADE_with_FastEmbed.ipynb

```diff
@@ -47,11 +47,25 @@
   },
```

Comment:
<p>Should we use <code>SparseTextEmbedding</code> as the type annotation instead of <code>SparseEmbedding</code> here? I'd prefer that to avoid importing that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qdrant/fastembed/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>SparseTextEmbedding and SparseEmbedding are different entities and not mutually exclusive, what do you mean here?</p>

Reply:
This is a separate question and I'll ask again after thinking a bit more about this.